README.md

```diff
@@ -85,34 +90,35 @@ information on OpenDDS compliance with the DDS specification. If you would like
 to contribute a feature or sponsor the developers to add a feature  please see
 the Support section above for contact information.
 
-
 ## Dependencies
 
 For a complete detailed list of dependencies, see
-[docs/dependencies.md](docs/dependencies.md).
+[`docs/dependencies.md`](docs/dependencies.md).
```

Comment:
Should this file contain a summary of the optional dependencies?  I think at least OpenSSL and Xerces-C++ should be mentioned since they are used in the core when security is enabled.  Same idea for JDK.  We could also list Qt, Wireshark, etc. without giving details here.

Reply:
Sure, having a miscellaneous section makes sense